src/web/src/components/Header/DesktopHeader.tsx

```diff
@@ -19,46 +22,50 @@ import Login from '../Login';
 
 const useStyles = makeStyles((theme) => ({
   root: {
+    width: '15em',
+    height: '32em',
+    flexDirection: 'column',
+    justifyContent: 'end',
+    position: 'fixed',
+    top: '0',
```

Comment:
`top: '1.4em'`, 

To achieve this result:
<img width="564" alt="Screen Shot 2021-02-24 at 1 24 13 AM" src="https://user-images.githubusercontent.com/60857409/108957192-0f800f00-763f-11eb-978c-fa8031d9d92f.png">